Source/WebCore/platform/graphics/filters/FilterOperation.cpp

```diff
@@ -91,7 +91,7 @@ void ReferenceFilterOperation::loadExternalDocumentIfNeeded(CachedResourceLoader
 {
     if (m_cachedSVGDocumentReference)
         return;
-    if (!SVGURIReference::isExternalURIReference(m_url, *cachedResourceLoader.document()))
+    if (!SVGURIReference::isExternalURIReference(m_url, Ref { *cachedResourceLoader.document() }))
```

Comment:
We should add the following getter to CachedResourceLoader and call it:
```
RefPtr<Document> protectedDocument() const { return document(); }
```

Note that I was doing the same as you in my earlier patches but Darin preferred using `the protectedX()` method pattern to avoid all these explicit `Ref { }`.